### DISCUSSION TITLE
How could I use this script in Fedora SilverBlue?

### DISCUSSION THREAD

**Discussion by Heus-Sueh** [prasanthrangan/hyprdots#659] [Ideas]
I wanted to be able to use this script in Fedora SilverBlue with Hyprland already installed, would there be any way to adapt this script to only copy the dotfiles and not install the packages related to Arch Linux?

**Comment by kRHYME7** [comment]
Yes, you can start looking at the ./restore_cfg.sh . or try to run  ``` ./install -r ```  but you should install all the needed packages as it won't copy the files if it is not installed.  see ``` ./restore_cfg.lst ``` last column indicates the needed package for the specific files
